dynawo/sources/Modeler/Common/DYNModeler.cpp

```diff
@@ -155,15 +155,12 @@ Modeler::initParamDescription(const shared_ptr<ModelDescription>& modelDescripti
         if (refType == "DOUBLE") {
           double value = data_->getStaticParameterDoubleValue(staticID, refOrigName);
           params->createParameter(*itRef, value);
-          Trace::info("MODELER") << DYNLog(AddedRefParToSet, modelDescription->getID(), *itRef, value) << Trace::endline;
         } else if (refType == "INT") {
           int value = data_->getStaticParameterIntValue(staticID, refOrigName);
           params->createParameter(*itRef, value);
-          Trace::info("MODELER") << DYNLog(AddedRefParToSet, modelDescription->getID(), *itRef, value) << Trace::endline;
         } else if (refType == "BOOL") {
           bool value = data_->getStaticParameterBoolValue(staticID, refOrigName);
           params->createParameter(*itRef, value);
-          Trace::info("MODELER") << DYNLog(AddedRefParToSet, modelDescription->getID(), *itRef, value) << Trace::endline;
         } else {
           throw DYNError(Error::MODELER, ParameterWrongTypeReference, *itRef);
         }
```

Comment:
I don't remember but I suppose it is handled by the Parameter log somewhere. Could you confirm?

Reply:
Not really, those are the parameters gathered in the IIDM and as you noticed earlier we mix them with the PAR ones. I plan to add again this source in another commit and then the parameters system will handle them automatically

Reply:
Fine